doc/plutus/tutorials/basic-apps.rst

```diff
@@ -137,7 +137,7 @@ All that's missing now is the code for retrieving the funds, and some glue to pu
 
 In ``unlockFunds`` you use the constraints library to build the spending transaction.
 Here, ``tx`` combines three different constraints.
-:hsobj:`Ledger.Tx.Constraints.collectFromTheScript` takes the script outputs in ``unspentOutputs`` and adds them as input to the transaction, using the unit ``()`` as the redeemer.
+`Ledger.Tx.Constraints.collectFromTheScript` [TODO broken link] takes the script outputs in ``unspentOutputs`` and adds them as input to the transaction, using the unit ``()`` as the redeemer.
```

Comment:
Please help: `Ledger.Tx.Constraints.collectFromTheScript` is obsolete. What should I replace it with?

Reply:
`Ledger.Tx.Constraints.spendUtxosFromTheScript`

Reply:
_note to self: refresh page before commenting_